packages/trader/src/Modules/MT5/Containers/mt5-dashboard.jsx

```diff
@@ -157,6 +162,23 @@ class MT5Dashboard extends React.Component {
                                 />
                             </div>
                         </Tabs>
+                        <div className='mt5-dashboard__info'>
+                            <div className='mt5-dashboard__info-description'>
+                                <Localize i18n_default_text='Use these in your apps' />
+                            </div>
+                            <MT5InfoCopy
+                                name={getBrokerName()}
+                                label={localize('Broker')}
+                                info_msg={localize('Click here to copy broker name.')}
+                                success_msg={localize('broker name copied!')}
```

Comment:
Should the text be `Broker` with capital `B`?

Reply:
Thanks, probably sped through the changes that I didn't notice. :D